extensions/wikia/ArticleVideoMVP/styles/premium-mvp.scss

```diff
@@ -0,0 +1,135 @@
+#premium-mvp-article-header {
+	display: block;
+}
+
+#premium-mvp-video {
+	background-position: center center;
+	background-size: cover;
+	cursor: pointer;
+	margin-bottom: 11px;
+	padding-top: 56%;
+	position: relative;
+	width: 100%;
+
+	&:before {
+		background-image: linear-gradient(to bottom, rgba(0, 0, 0, 0), #000000);
```

Comment:
It's a good practice to use variables here.Especially when something depends on wiki colors.